benchmark.sh

```diff
@@ -16,7 +16,7 @@ trap cleanup EXIT SIGINT SIGTERM
 # Generate random numbers for insert and find operations
 INSERT_FILE="inserts.json"
 FIND_FILE="finds.txt"
-NUM_ENTRIES=100000
+NUM_ENTRIES=100
```

Comment:
Overall looks good to me.
Is there a particular reason you changed the number of entries here?